docs/admin_guide/french_specifities/assets/dcat_data_gouv_fr_output.patch

```diff
@@ -0,0 +1,307 @@
+ 
+     <dct:abstract>
+       <xsl:value-of select="gmd:abstract/gco:CharacterString"/>
++
++      <!-- SIB addon-->
++      <!-- Layout will be visible only in source mode in the browser. If not in source mode, newlines won't be displayed and the keywords will be shown inline -->
++      <xsl:text>&#xa;</xsl:text> <!-- linebreak -->
++      <!-- Keywords -->
++      <xsl:for-each-group select="//gmd:MD_Keywords[(gmd:thesaurusName)]/gmd:keyword"
++                          group-by="gco:CharacterString|gmx:Anchor">
++        <xsl:text>&#xa;</xsl:text> <!-- linebreak -->
++        <xsl:value-of select="normalize-space(../gmd:thesaurusName/*/gmd:title)"/> : <xsl:value-of select="normalize-space(gco:CharacterString|gmx:Anchor)"/><xsl:text>.</xsl:text>
++      </xsl:for-each-group>
+     </dct:abstract>
+     <!-- xpath: gmd:identificationInfo/*/gmd:abstract/gco:CharacterString -->
+ 
+@@ -324,9 +352,13 @@
+     <!-- "The main category of the dataset. A dataset can have multiple themes."
+     -->
+     <xsl:for-each
+-      select="gmd:descriptiveKeywords/gmd:MD_Keywords[(gmd:thesaurusName)]/gmd:keyword">
++      select="gmd:descriptiveKeywords/gmd:MD_Keywords[(gmd:thesaurusName)]/gmd:keyword/gco:CharacterString">
+       <dcat:theme
+-        rdf:resource="{iso19139:getKeywordURI(., iso19139:getThesaurusURI(../gmd:thesaurusName, $resourcePrefix))}"/>
++        rdf:resource="{iso19139:getKeywordURI(.., iso19139:getThesaurusURI(../../gmd:thesaurusName, $resourcePrefix))}"/>
++    </xsl:for-each>
++    <xsl:for-each
++      select="gmd:descriptiveKeywords/gmd:MD_Keywords[(gmd:thesaurusName)]/gmd:keyword/gmx:Anchor">
++      <dcat:theme rdf:resource="{@xlink:href}" />
+     </xsl:for-each>
+ 
+     <xsl:for-each select="gmd:topicCategory/gmd:MD_TopicCategoryCode[.!='']">
+@@ -365,13 +397,12 @@
+     <!-- "The temporal period that the dataset covers." -->
+     <!-- TODO could be improved-->
+     <xsl:for-each
+-      select="gmd:extent/*/gmd:temporalElement/gmd:EX_TemporalExtent/gmd:extent/(gml:TimePeriod|gml320:TimePeriod)">
++      select="gmd:extent/*/gmd:temporalElement/gmd:EX_TemporalExtent/gmd:extent/gml:TimePeriod">
+       <dct:temporal>
+-        <xsl:value-of select="gml:beginPosition|gml320:beginPosition"/>
+-        <xsl:if test="gml:endPosition|gml320:endPosition">
+-          /
+-          <xsl:value-of select="gml:endPosition|gml320:endPosition"/>
+-        </xsl:if>
++        <xsl:choose>
++          <xsl:when test="gml:endPosition != ''"><xsl:value-of select="gml:beginPosition"/>/<xsl:value-of select="gml:endPosition"/></xsl:when>
++          <xsl:otherwise><xsl:value-of select="substring(gml:beginPosition, 1, 7)"/></xsl:otherwise>
++        </xsl:choose>
+       </dct:temporal>
+     </xsl:for-each>
+     <!-- xpath: gmd:identificationInfo/*/gmd:extent/*/gmd:temporalElement -->
+@@ -397,10 +428,47 @@
+ 
+ 
+     <!-- "The frequency with which dataset is published." See placetime.com intervals. -->
++    <!-- SIB addon-->
```

Comment:
Replace SIB addon by geOrchestra addition:

Reply:
Maybe mention that this is a partial backport from https://github.com/geonetwork/core-geonetwork/pull/7600